src/FieldServiceProvider.php

```diff
@@ -19,7 +19,16 @@ class FieldServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->app['router']->pushMiddlewareToGroup('nova', InterceptFlexibleAttributes::class);
+        $router = $this->app['router'];
+        if ($router->hasMiddlewareGroup('nova')) {
+            $this->app['router']->pushMiddlewareToGroup( 'nova', InterceptFlexibleAttributes::class );
+        }
+        elseif (! $this->app->configurationIsCached()) {
+            $this->app['config']->set('nova.middleware', array_merge(
+                $this->app['config']->get('nova.middleware', []),
```

Comment:
Could this be more concise? Correct me if I'm wrong but couldn't this be simply written as `config('nova.middleware')`?

Reply:
True! I was inspired by `ServiceProvider::mergeConfigFrom()` which is why my code was a bit more verbose.